java/ehcache/Portfile

```diff
@@ -1,53 +1,62 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
 PortSystem 1.0
+PortGroup               java 1.0
+PortGroup               github 1.0
+
+github.setup            ehcache ehcache3 3.8.1 v
+github.tarball_from releases
+name                    ehcache
+distname                ${name}-${version}
+
+categories              java
+maintainers             nomaintainer
+platforms               darwin
+license                 Apache-2
+
+description             Ehcache is a pure Java, in-process object cache.
+long_description        Ehcache is a pure Java, in-process cache with the \
+                        following features: \
+                        1.  Fast \
+                        2.  Simple \
+                        3.  Acts as a pluggable cache for Hibernate \
+                        4.  Small foot print. Both in terms of size and memory \
+                        requirements. \
+                        5.  Minimal dependencies. \
+                        6.  Fully documented. See the online Documentation, FAQ \
+                        and the online JavaDoc. \
+                        7.  Comprehensively Test Coverage. See the clover test \
+                        report. \
+                        8.  Scalable to hundreds of caches and large multi-cpu \
+                        servers. \
+                        9.  Provides LRU, LFU and FIFO cache eviction policies. \
+                        10. Available under the Apache 1.1 license. Ehcache's \
+                        copyright and licensing has been reviewed and approved \
+                        by the Apache Software Foundation, making ehcache \
+                        suitable for use in Apache projects. \
+                        11. Production tested. All final versions of ehcache \
+                        are production tested for several weeks on a large and \
+                        very busy eCommerce site before release.
```

Comment:
You are wasting too much space here. Move these lines left to start at column 4. Then you can re-wrap out to column 80. Then it won't be using as many lines. Does that make sense?

Reply:
Hi Sir,
I've made the change accordingly. Please review my latest commit. Thanks.